docs/pages/router/advanced/static-redirects.mdx

```diff
@@ -0,0 +1,148 @@
+---
+title: Redirects and rewrites
+description: Learn how to create redirects for routes that may no longer exist
+---
+
+> **warning** Static redirects were added Expo Router `4.x.x`.
+
+## Redirect component
+
+To use <Redirect />, simply import it from expo-router and specify the href prop with the desired destination route.
+
+```tsx
+import { Redirect } from 'expo-router';
+
+export default function Index() {
+  return <Redirect href="/home" />;
+}
+```
+
+In this example, when the user visits this page, they will be automatically redirected to /home.
+
+```tsx
+You can use <Redirect /> within a component that checks for conditions before navigating.
+
+import { Redirect } from 'expo-router';
+import { useState, useEffect } from 'react';
+
+export default function ProtectedPage() {
+  const [isAuthenticated, setIsAuthenticated] = useState(false);
+
+useEffect(() => {
+  // Simulate checking authentication status
+  setTimeout(() => setIsAuthenticated(true), 2000);
+}, []);
+
+if (!isAuthenticated) {
+  return <Redirect href="/login" />;
+}
+
+  return <Text>Welcome to the protected page!</Text>;
+}
+```
+
+In this case, if the user is not authenticated, they will be redirected to the /login page.
+
+## Static redirects
+
+Static redirects allow you to specify a redirect configuration in your **app.json**. This allows you to specify redirects for routes that may no longer exist in your **app** directory. When you navigate to a screen, Expo Router simulates a `GET` request to a local server which responds with the screens contents. This ensures that Expo Router behaves consistently on native, web and when fetching React Server Components. As such, this configuration is based upon **server requests** and is based upon the request URL, not the screen name.
+
+```json app.json
+{
+  "plugins": [
+    [
+      "expo-router",
+      "redirects": [{
+        "source": "/redirect/from/here",
+        "destination: "/to/this/route"
+      }]
+    ]
+  ]
+}
+```
+
+Each redirect configuration should have the following properties:
+
+- **source** is the incoming request path pattern.
+- **destination** is the path you want to route to.
+
+And can have these optional properties:
+
+- **permanent** true or false - if true will use the 308 status code which instructs clients/search engines to cache the redirect forever, if false will use the 307 status code which is temporary and is not cached.
```

Comment:
```suggestion
- **permanent**: Accepts `true` or `false`. If `true`, will use the 308 status code, which instructs clients/search engines to cache the redirect forever. If `false`, will use the 307 status code, which is temporary and is not cached.
```

Let's add clarity by using `,` to pause property, and using code syntax highlights for boolean values.